dotnet/IoTSub.cs

```diff
@@ -0,0 +1,6 @@
+mqttClient.Subscribe("device:rocket:1");
+mqttClient.On("message", (topic, message) => {
+  if (topic == "action") {
+    rocket.ProcessInstruction(JSON.Parse(message));
```

Comment:
We don't have a helper called `JSON` but if it doesn't need to compile then it does convey what needs to happen.

Reply:
I think it's copy pasted from javascript, there we have native `JSON.Parse` native support to parse JSON

Reply:
> We don't have a helper called `JSON` but if it doesn't need to compile then it does convey what needs to happen.

Exactly. To fit in the space provided, we pretty much have to employ what I hoped where reasonable fictional external types.

Reply:
Resolving this conversation so that we can land this work as it's needed for other teams to work on tomorrow.